### ISSUE TITLE
Cannot list users and organizations?

### ISSUE THREAD

**Issue by Aracki** [esnet/gdg#184]
Although my service account has admin rights I cannot list users/orgs.

I can list all other objects and teams as well.

Getting these errors:
```
response error for https://grafana.tenderly.link/api/users?page=1&perpage=5000: unexpected status: 403

error="[GET /orgs][403] searchOrgsForbidden  &{Error: Message:0x140003609a0 Status:}"
```

**Comment by safaci2000** [comment]
Hello @Aracki.  

A few things, what version of GDG are you running?  I have done some rework for orgs in master but I'm going to assume you're running the latest release. 

So for the Grafana API in general there's what I'd call 3 tiers of permissions. 

1. Token based (Most entities you can do basic CRUD with admin tokens and allow you update/delete/etc.. items)
2. BasicAuth (Some entities require Basic Auth, doesn't matter if the token has admin permissions is still needs a username/password to work).
3. GrafanaAdmin for doing things with Orgs and such you basically need super admin.  Even if the user is an Grafana Admin that created the token it still isn't enough.  You need Basic Auth with Grafana Admin permissions. (note, it doesn't HAVE to be your default 'admin' user, but it has to be user/password auth with the same level or privileges).

I'm actively working on some rework on Orgs tha'll change the behavior a good bit for orgs, but for the short version.

TLDR: if you are need to list orgs, you need to be a GrafanaAdmin and use basic auth (user/password).
